src/modules/organization/createOrg/create-org-success-modal/CreateOrgSuccessModal.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <base-modal :maxModalWidth="'570px'" :isClosable="false"
+    ><div class="flex flex-col items-center">
+      <base-icon :width="85" :height="85" class="text-green-5 mb-4"
+        ><CheckCircleIcon
+      /></base-icon>
+      <h1 class="font-heading text-3xl mb-2">
+        Your team has been created!
+      </h1>
+      <p class="text-gray-5 text-center mb-7 leading-relaxed">
+        Your organization has been created.<br />
+        To be a verified organization, verification is needed.<br />
+        Please contact <span class="text-primary">admin@onepass.com</span> for

```

Comment:
Considering using `<a href="mailto:example@somewhere.com">Send email to example@somewhere.com</a>`. This will enable email app to works with the link